src/io/result_io.jl

```diff
@@ -369,13 +369,14 @@ function setup_result_store!(domain::Domain, scen_spec::DataFrame)::Tuple
     mkdir(joinpath(log_location, SPATIAL_DATA))
     geo_fn = joinpath(log_location, SPATIAL_DATA, basename(attrs[:name]) * ".gpkg")
     try
-        GDF.write(geo_fn, domain.site_data; driver="geopackage")
+        GDF.write(geo_fn, domain.site_data; driver="GPKG")
     catch err
         if !isa(err, ArgumentError)
             rethrow(err)
         end
 
-        GDF.write(geo_fn, domain.site_data; geom_columns=(:geom,), driver="geopackage")
+        col = _get_geom_col(domain.site_data)
```

Comment:
Should we move this into the main package?
```
ERROR: UndefVarError: `_get_geom_col` not defined
```

Reply:
Ugh, my bad again. Must've had a global copy loaded. I'll sort it out